packages/web3-eth/test/integration/web3_eth/send_transaction.test.ts

```diff
@@ -0,0 +1,121 @@
+			const transaction: Transaction = {
+				from: accounts[0],
+				data: greeterContractDeploymentData,
+				gas: '0x740b8',
+			};
+			const response = await web3Eth.sendTransaction(transaction);
+			expect(response.status).toBe('0x1');
```

Comment:
we should also check in mined tx, provided `from` and `data` match. For verification that its not modified in our lib or any depend libs. Also check that value is 0.

Reply:
Addressed by [this](https://github.com/ChainSafe/web3.js/pull/5056/commits/290b8b3a02da93af8f91eb4b7d39b7938b762063) commit